setup.py

```diff
@@ -8,15 +9,38 @@
 except ImportError:
     pass
 
-import setuptools
+from setuptools import setup, Extension
 from setuptools.command.build_ext import build_ext as _build_ext
 
 
-with open('README.rst') as f:
-    readme_content = f.read()
+MAJOR = 0
+MINOR = 2
+MICRO = '0.dev0'
```

Comment:
Can we single source the version the same way as pymongocrypt does it? https://github.com/mongodb/libmongocrypt/blob/master/bindings/python/setup.py#L37

Reply:
Done!